openfe/tests/protocols/restraints/test_geometry_boresch_host.py

```diff
@@ -142,13 +148,114 @@ def test_evaluate_host2_good(eg5_protein_ligand_universe):
+                universe.atoms[host_anchor[j]].position,
+                box=universe.dimensions,
+            )
+            assert dist == pytest.approx(ref)
+
+    def test_not_collinear(self, host_anchor, universe):
+        # check none of the g2-g1-g0-h0-h1-h2 vectors are not collinear
+        assert not is_collinear(
+            positions=np.vstack((
+                universe.atoms[[5528, 5507, 5508]].positions,
+                universe.atoms[host_anchor].positions
+            )),
+            atoms=[0, 1, 2, 3, 4, 5],
+            dimensions=universe.dimensions
+        )
+
+    def test_angles(self, host_anchor, universe):
+        # check that the angles aren't flat
+        ag1 = mda.lib.distances.calc_angles(
+            universe.atoms[5507].position,
+            universe.atoms[5528].position,
+            universe.atoms[host_anchor[0]].position,
+            box=universe.dimensions
+        )
+        ag2 = mda.lib.distances.calc_angles(
+            universe.atoms[5528].position,
+            universe.atoms[host_anchor[0]].position,
+            universe.atoms[host_anchor[1]].position,
+            box=universe.dimensions
+        )
+        for angle in [ag1, ag2]:
+            assert check_angle_not_flat(
+                angle=angle * unit.radians,
+                force_constant=83.68 * unit.kilojoule_per_mole / unit.radians**2,
+                temperature=298.15 * unit.kelvin,
+            )
+
+    def test_dihedrals(self, host_anchor, universe):
+        dih1 = mda.lib.distances.calc_dihedrals(
+            universe.atoms[5508].position,
+            universe.atoms[5507].position,
+            universe.atoms[5528].position,
+            universe.atoms[host_anchor[0]].position,
+            box=universe.dimensions,
+        )
+        dih2 = mda.lib.distances.calc_dihedrals(
+            universe.atoms[5507].position,
+            universe.atoms[5528].position,
+            universe.atoms[host_anchor[0]].position,
+            universe.atoms[host_anchor[1]].position,
+            box=universe.dimensions,
+        )
+        dih3 = mda.lib.distances.calc_dihedrals(
+            universe.atoms[5528].position,
+            universe.atoms[host_anchor[0]].position,
+            universe.atoms[host_anchor[1]].position,
+            universe.atoms[host_anchor[2]].position,
+            box=universe.dimensions,
+        )
+        assert check_dihedral_bounds(dih1 * unit.radians)
+        assert check_dihedral_bounds(dih2 * unit.radians)
+        assert check_dihedral_bounds(dih3 * unit.radians)
+
 @pytest.mark.slow
 def test_find_host_anchor_none(eg5_protein_ligand_universe):
 
     host_anchor = find_host_anchor(
         guest_atoms=eg5_protein_ligand_universe.atoms[[5528, 5507, 5508]],
         host_atom_pool=eg5_protein_ligand_universe.select_atoms("backbone"),
-        minimum_distance=4.5 * unit.nanometer,
+        host_minimum_distance=4.5 * unit.nanometer,
+        guest_minimum_distance=4.5 * unit.nanometer,
```

Comment:
does this return none due to both of these cutoffs being >>2? Maybe a docstring here.